yoti/tests/yoti.test

```diff
@@ -153,6 +167,37 @@ class YotiTest extends DrupalWebTestCase {
       $attribute_values,
       'Check button markup is correct.'
     );
+
+    // Check that SDK script is included on page.
+    $this->assertElementByXpath(
+      '//head/script[@src=:src]',
+      array(
+        ':src' => 'https://sdk.yoti.com/clients/browser.2.1.0.js',
```

Comment:
I know it's only in our tests, but is there a way of ignoring the version number on the end, so we won't have to update this when we update the browser.js?

Reply:
Fixed in 75961ca